refinery/templates/base.html

```diff
@@ -322,6 +322,9 @@
   <script src="{% static "vendor/jquery-file-upload/js/jquery.fileupload-angular.js" %}"></script>
   <script src="{% static "js/data-set-import.js" %}"></script>
 
+  <!-- Fetch external SVGs referenced in <use> elements when the browser fails to do so -->
+  <script defer src="{% static "vendor/svgxuse/svgxuse.js" %}"></script>
```

Comment:
Does this work? It looks like you want to load the lib async (i.e., `defer`) after other JS stuff. What happens if the icons are suppose to be loaded before the lib is loaded?